docs/buildah.md

```diff
@@ -26,7 +26,19 @@ Print debugging information
 
 **--default-mounts-file**
 
-Path to default mounts file (default path: "/usr/share/containers/mounts.conf")
+Path to default mounts file (default path: "/usr/share/containers/mounts.conf").  Users can also create an additional conf file, `/etc/containers/mounts.conf`.
+
+The mounts.conf files specify volume mount directories that are automatically mounted inside containers when executing the `buildah run` or `buildah build-using-dockerfile` commands.  Container process can then use this content.  The volume mount content does not get committed to the final image.
+
+Usually these directories are used for passing secrets or credentials required by the package software to access remote package repositories.  For example a mounts.conf with the `/usr/share/rhel/secrets:/run/secrets like, the content of /usr/share/rhel/secrets directory is mounted on /run/secrets inside the container.  This allows Red Hat Enterprise Linux subscriptions from the host to be used
```

Comment:
"For example a mounts.conf with the `/usr/share/rhel/secrets:/run/secrets like," .  You're missing a back tick  and I think you wanted line not like?

Perhaps "For example a mounts.conf with the line `/usr/share/rhel/secrets:/run/secrets`, "

Reply:
Fixed.